pkg/operator/workload/workload_openshiftapiserver_v311_00_sync.go

```diff
@@ -345,6 +345,7 @@ func manageOpenShiftAPIServerDeployment_v311_00_to_latest(
 		resourcehash.NewObjectRef().ForConfigMap().InNamespace(operatorclient.TargetNamespace).Named("etcd-serving-ca"),
 		resourcehash.NewObjectRef().ForConfigMap().InNamespace(operatorclient.TargetNamespace).Named("image-import-ca"),
 		resourcehash.NewObjectRef().ForConfigMap().InNamespace(operatorclient.TargetNamespace).Named("trusted-ca-bundle"),
+		resourcehash.NewObjectRef().ForConfigMap().InNamespace(operatorclient.TargetNamespace).Named("openshift-apiserver-audit-policies"),
```

Comment:
I think we don't' have to calculate the hash since the cm will be revisioned anyway.